app/models/certificate.rb

```diff
@@ -82,6 +86,10 @@ def for_surveys(surveys)
     end
   end
 
+  def init
+    self.attained_level ||= 'none'
+  end
+
```

Comment:
Oh really? That's good. Maybe it changed at some point, or maybe I'm imagining things. I'll stick it in a migration then.